src/Chunk.ts

```diff
@@ -977,10 +974,10 @@ export default class Chunk {
 
 	private computeChunkName(): string {
 		if (this.facadeModule !== null && this.facadeModule.chunkAlias) {
-			return this.facadeModule.chunkAlias;
+			return sanitizeFileName(this.facadeModule.chunkAlias);
```

Comment:
It could be good to ensure that `chunkAlias` is sanitized at the time it is set on the modules, instead of at this stage. Strictly speaking, we should probably throw for invalid scenarios over sanitizing as this is user input? But it's very much a nit.

Reply:
In fact no, we should not sanitize the id on the module. The reason is that by convention, some plugins (such as rollup-plugin-multi-entry) create virtual modules with an id that starts with a `\0` to signal that other plugins should ignore this module for e.g. loading and transformation. Thus we need to keep the module id unsanitized to not break plugins. The chunk id, however, is what is written to disk and is something we can choose rather freely.